projects/laji/src/app/shared-modules/datatable/datatable/datatable.component.ts

```diff
@@ -389,12 +389,6 @@ export class DatatableComponent implements AfterViewInit, OnInit, OnChanges, OnD
   }
 
   private defaultSort(rows: any[]): any[] {
```

Comment:
Would be trivial to add generics here
```
  private defaultSort<T>(rows: T[]): T[] {
    return [...rows].sort((a: T, b: T) => a.preSortIndex < b.preSortIndex ? -1 : 1);
  }
```

Reply:
It can't be a generic type because of the `preSortIndex` prop but I added a new type `DatatableRow`